frontend/src/component/input/NumberInput.vue

```diff
@@ -20,8 +20,18 @@ const value = ref(props.value)
 watch(() => props.value, (newValue) => {
     value.value = newValue
 })
+function checkEmptyInput () {
```

Comment:
Deze functie naam is beetje verwarrend, als je naar de naam kijkt verwacht je dat ie alleen iets gaat checken maar hij roept de update functie aan.

Is deze ook niet dubbel met de check in update zelf?

Reply:
Betere naam zou iets zijn zoals updateIfValid